frontend/packages/component-library/src/tags/stories/Tags.story.tsx

```diff
@@ -23,6 +24,41 @@ const SingleTemplate: StoryFn<TagsProps> = args => (
   </>
 );
 
+const SingleTemplateWithTagState: StoryFn<TagsProps> = args => {
+  const initialState = ['AAA', 'BBB', 'CCC'];
+  const [tags, setTags] = useState(initialState);
+
+  const removeTag = (index: number) => {
+    setTags(prev => {
+      const newTags = [...prev];
+      newTags.splice(index, 1);
+      return newTags;
+    });
+  };
+
+  const tagsList: TagProps[] = tags.map((label, i) => ({
+    label,
+    tooltipId: label,
+    tooltipContent: label,
+    onClose: () => removeTag(i),
```

Comment:
nit: we now need to add `type: 'closable'` here to make this work (checked locally)

```suggestion
    onClose: () => removeTag(i),
    type: 'closable',

Reply:
oof! thanks!